src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 4.2.0
 -----
 
+* added a `supportsClass()` check prior to `refreshUser()` for each user provider
```

Comment:
Should be reverted, if this is a bugfix it should be based off the `3.4` branch. And for bugfixes we do not add entries to the components changelog files. If this is a new feature, it will be part of 4.3.